examples/pybullet/gym/pybullet_envs/deep_mimic/env/humanoid_stable_pd.py

```diff
@@ -893,7 +899,22 @@ def getReward(self, pose):
           linkStateSim = self._pybullet_client.getLinkState(self._sim_model, j)
           linkStateKin = self._pybullet_client.getLinkState(self._kin_model, j)
         linkPosSim = linkStateSim[0]
+        linkOrnSim = linkStateSim[1]
         linkPosKin = linkStateKin[0]
+        linkOrnKin = linkStateKin[1]
+        
+        # important: change the reference frames of the link pos/orn
+        # to models' local reference frames, compare in this coord system
+        # this replicates functionality in original code
+        linkPosSim, linkOrnSim = self._pybullet_client.multiplyTransforms(
+          simRootTransPos, simRootTransOrn,
+          linkPosSim, linkOrnSim
+        )
+        linkPosKin, linkOrnKin = self._pybullet_client.multiplyTransforms(
+          kinRootTransPos, kinRootTransOrn,
+          linkPosKin, linkOrnKin
+        )
+        
```

Comment:
This needs to be tested. This replicates the following lines in Jason Peng's code, which transform the link positions to the body-local reference frame:
https://github.com/xbpeng/DeepMimic/blob/687945c8abde39fd24e1d175cc155f7cb8845729/DeepMimicCore/scenes/SceneImitate.cpp#L91-L92